src/Blocks/BlockPiston.cpp

```diff
@@ -55,50 +61,46 @@ void cBlockPistonHandler::ConvertToPickups(cItems & a_Pickups, NIBBLETYPE a_Bloc
 
 
 
-
-bool cBlockPistonHandler::GetPlacementBlockTypeMeta(
-	cChunkInterface & a_ChunkInterface, cPlayer * a_Player,
-	int a_BlockX, int a_BlockY, int a_BlockZ, eBlockFace a_BlockFace,
-	int a_CursorX, int a_CursorY, int a_CursorZ,
-	BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta
-)
+bool cBlockPistonHandler::GetPlacementBlockTypeMeta(cChunkInterface  & a_ChunkInterface, cPlayer * a_Player, int a_BlockX, int a_BlockY, int a_BlockZ, eBlockFace a_BlockFace, int a_CursorX, int a_CursorY, int a_CursorZ, BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta)
```

Comment:
Line too long, the old style was much better. Please don't make changes like this.